react/README.md

```diff
@@ -386,7 +386,7 @@
 
     // good
     <Foo
-      ref={(ref) => { this.myRef = ref; }}
+      ref={ref => this.myRef = ref}
```

Comment:
According to the official React documentation (https://facebook.github.io/react/docs/refs-and-the-dom.html) it's a totally valid approach:

> There is even a shorter way to write it: ref={input => this.textInput = input}.

But ofc., if you want me to change it back I can do so!